app/models/abstract_contributor.rb

```diff
@@ -5,8 +5,7 @@
 # It is necessary so that Rails STI will assign the proper type to each concrete model
 # and so that when you query Work#contributors you only get the non-author contributors.
 class AbstractContributor < ApplicationRecord
-  self.abstract_class = true
-  self.table_name = 'contributors'
+  # self.abstract_class = true
```

Comment:
Cruft? Or do you mean to uncomment this again?

Reply:
Thanks, I've removed this cruft.